website/src/controllers/user.js

```diff
@@ -117,12 +117,26 @@ api.score = function(req, res, next) {
     }, saved.toJSON().stats));
 
     // Webhooks
+
+    // Select character data to send
+    var userData = _.pick(user.toJSON(), ['_id', '_tmp', 'stats']); // user.toJSON to copy-by-value
```

Comment:
We could just add `toNextLevel`, `maxHealth`, and `maxMP` to the `userData` object instead of the child-object `stats`, but then this wouldn't be consistent with the `getUser` endpoint.